docs/manual_testing/app_plugin/index.rst

```diff
@@ -0,0 +1,11 @@
+Testing Kolibri with app plugin enabled
+=============
```

Comment:
```
/home/runner/work/kolibri/kolibri/docs/manual_testing/app_plugin/index.rst:2:Title underline too short.
```

Need to check the docs build! It may also complain about the underline being too long below.